### PR TITLE
#135593037 The Front tests need fixing

### PR DESCRIPTION
**Pull request by RyanSept**
#### What does this PR do?
Adds tests for the front end specifically for the integrations/channels and checks features.
#### Description of Task to be completed?
Test if team access works for the channel/ and checks feature
#### How should this be manually tested?
Run `python manage.py test`
#### What are the relevant pivotal tracker stories?
#135593037

### CODE REVIEW

**Review by jubrilissa — COMMENTED** [review]
Why do we have database.json created?

**Review by jubrilissa — CHANGES_REQUESTED** [review]
Why do we have `setting copy.py` here?
Also, avoid file names with spaces

**Comment by RyanSept** [comment]
@andela-ijubril I created database.json as a backup for when I wanted to flush the sqlite db. I however ended up not doing this but forgot to remove the file. 
I created setting copy.py as a backup too when switching the database from sqlite to postgres. Thank you, I shall make an effort to avoid spaces in filenames; it was a Windows OS habit.

File: manage.py [hunk 1]
```diff
@@ -6,5 +6,6 @@
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hc.settings")
 
     from django.core.management import execute_from_command_line
-
+    from django.db import connection
+    print "***",connection.vendor
```
Comment (jubrilissa) [inline manage.py:10]: Is there any reason why we also have this here?

**Pull request closed without merging**